src/EventSubscriber/AddEditOrderActionSubscriber.php

```diff
@@ -9,19 +9,14 @@
 use Sylius\Component\Grid\Event\GridDefinitionConverterEvent;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-/**
```

Comment:
What's wrong with this? Too obvious? :)

Reply:
Imo yest, what else can the `AddEditOrderActionSubscriber` do? 😄

Reply:
Haha, you're right